spec/javascripts/Navigation/header/AccountSettingsMenu.spec.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import Enzyme, { shallow } from 'enzyme'
+import Adapter from 'enzyme-adapter-react-16'
+import { AccountSettingsMenu } from 'Navigation/components/header/AccountSettingsMenu'
+
+Enzyme.configure({ adapter: new Adapter() })
+
+const props = {
+  accountSettingsLink: 'my-link',
+  accountSettingsClass: 'my-class'
+}
+
+describe('<AccountSettingsMenu/>', () => {
```

Comment:
From a presentational point of view it's a bit redundant to `describe` the file being tested:
```
 PASS  spec/javascripts/Navigation/header/AccountSettingsMenu.spec.jsx
  <AccountSettingsMenu/>
    ✓ renders <AccountSettingsMenu/> component (18ms)
```
Usually describe is used to separate blocks of functionality. For instance `AccountSettingsMenu` could be open or close, then:
```js
describe('when dropdown is displayed', () => {
  it('should hide the dropdown when clicked')
}
describe('when dropdown is hidden', () => {
  it('should display a dropdown when clicked')
}
```
Otherwise it's enough with simply using `it`.